frame/assets-registry/src/lib.rs

```diff
@@ -73,6 +73,11 @@ pub mod pallet {
 		ForeignAdminApproved,
 	}
 
+	#[derive(Debug, Clone, Copy, PartialEq, Eq, Encode, Decode, TypeInfo)]
+	pub struct Metadata {
```

Comment:
can we renamte it to `RemoteMetadata`? Because locally we have 12 for all currencies. Can you comment on what decimals are used on Hydra/Akala/Moonriver? Just to make sure these are different from ours 12?

Reply:
Renamed to `Foreign Metadata`, because we already use `ForeignAdmin`, `ForeignAssetId` and etc.

Moonriver uses 18 decimals of precision (https://github.com/PureStake/moonbeam/blob/master/runtime/moonriver/src/lib.rs#L117).
Acala uses 12 decimals of precision (https://github.com/AcalaNetwork/Acala/blob/master/modules/evm/src/lib.rs#L1117).
Acala, HydraDX, Moonriver support assets with various precision in their assets-registry.